docs/syntax.md

```diff
@@ -3,10 +3,10 @@ Below you will find the step syntax next to the name of the method it utilizes.
 
 **Kubernetes related steps**:
 1. 	`<GK> a Kubernetes cluster` [AKubernetesCluster](https://godoc.org/github.com/keikoproj/kubedog/pkg/kubernetes#Client.AKubernetesCluster)
-2.	`<GK> [I] <create|submit|delete> [the] resource <filename>.yaml` [ResourceOperation](https://godoc.org/github.com/keikoproj/kubedog/pkg/kubernetes#Client.ResourceOperation)
-3.	`<GK> [I] <create|submit|delete> [the] resource <filename>.yaml in [the] <namespacename> namespace` [ResourceOperationInNamespace](https://godoc.org/github.com/keikoproj/kubedog/pkg/kubernetes#Client.ResourceOperationInNamespace)
-4.	`<GK> [I] <create|submit|delete> [the] resources in <filename>.yaml` [MultiResourceOperation](https://godoc.org/github.com/keikoproj/kubedog/pkg/kubernetes#Client.MultiResourceOperation)
-5.	`<GK> [I] <create|submit|delete> [the] resources in <filename>.yaml in [the] <namespacename> namespace` [MultiResourceOperationInNamespace](https://godoc.org/github.com/keikoproj/kubedog/pkg/kubernetes#Client.MultiResourceOperationInNamespace)
+2.	`<GK> [I] <create|submit|delete|update> [the] resource <filename>.yaml` [ResourceOperation](https://godoc.org/github.com/keikoproj/kubedog/pkg/kubernetes#Client.ResourceOperation)
+3.	`<GK> [I] <create|submit|delete|update> [the] resource <filename>.yaml in [the] <namespacename> namespace` [ResourceOperationInNamespace](https://godoc.org/github.com/keikoproj/kubedog/pkg/kubernetes#Client.ResourceOperationInNamespace)
+4.	`<GK> [I] <create|submit|delete|update> [the] resources in <filename>.yaml` [MultiResourceOperation](https://godoc.org/github.com/keikoproj/kubedog/pkg/kubernetes#Client.MultiResourceOperation)
+5.	`<GK> [I] <create|submit|delete|update> [the] resources in <filename>.yaml in [the] <namespacename> namespace` [MultiResourceOperationInNamespace](https://godoc.org/github.com/keikoproj/kubedog/pkg/kubernetes#Client.MultiResourceOperationInNamespace)
 6.	`<GK> [the] resource <filename> should be <created|deleted>` [ResourceShouldBe](https://godoc.org/github.com/keikoproj/kubedog/pkg/kubernetes#Client.ResourceShouldBe)
```

Comment:
Do we also need a validation step to make sure the resources was updated? Although I am not sure if that would be straight forward.

Reply:
I think I would personally get more confidence from a custom step to assert the changes went through/the changes are working as expected. It might be difficult to capture a generic update validation. However, maybe we can add some common update patterns to kubedog as we discover them. WDYT?

Reply:
Sounds good